app/pages/onboarding/03-restore-wallet/restore-wallet.tsx

```diff
@@ -88,7 +91,7 @@ export const RestoreWallet: React.FC = () => {
           <ErrorText>{error}</ErrorText>
         </ErrorLabel>
       )}
-      {mnemonicLength === 12 && (
+      {mnemonicLength === 12 && hasSubmitted && (
```

Comment:
I think a better pattern would be to check for an error, rather than if it's of a specific length

Reply:
This isn't an error, though. It's just showing a warning.

Reply:
ah, ok then!